marklogic/tests/conftest.py

```diff
@@ -0,0 +1,71 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+from typing import Any, Dict, Generator
+
+import pytest
+import requests
+
+from datadog_checks.dev import docker_run
+from datadog_checks.dev.conditions import CheckDockerLogs, WaitFor
+
+from .common import ADMIN_PASSWORD, ADMIN_USERNAME, API_URL, CHECK_CONFIG, HERE, PASSWORD, USERNAME
+
+
+@pytest.fixture(scope="session")
+def dd_environment():
+    # type: () -> Generator[Dict[str, Any], None, None]
+
+    # Standalone
+    compose_file = os.path.join(HERE, 'compose', 'standalone/docker-compose.yml')
+    with docker_run(
+        compose_file=compose_file,
+        conditions=[CheckDockerLogs(compose_file, r'Deleted'), WaitFor(setup_admin_user), WaitFor(setup_datadog_user)],
+    ):
+        yield CHECK_CONFIG
+
+    # Cluster (works locally but not on CI)
```

Comment:
I think this should be clean up for now and maybe replaced by a comment saying Cluster support is needed.